salt/modules/ansiblegate.py

```diff
@@ -215,13 +215,13 @@ def __virtual__():
     ret = ansible is not None
     msg = not ret and "Ansible is not installed on this system" or None
     if msg:
-        log.warning(msg)
+        log.debug(msg)
```

Comment:
Actually, we shouldn't be logging in `__virtual__()` at all. In the past, this was not well-documented, but I just added some docs for this the other day in PR #44588. We either need to not log in here at all, or this needs to be at the `trace` level.